data/sql/updates/pending_db_world/rev_1679067456689625500.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `creature` WHERE (`id1` = 23272) AND (`guid` IN (2196));
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(2196, 23272, 0, 0, 530, 0, 0, 1, 1, 0, -2139.45, 5554.06, 50.29, 0.433, 25, 0, 0, 5589, 3155, 0, 0, 0, 0, '', 0);
```

Comment:
Is this sniffed?

Reply:
yes

Reply:
Provide the sniff.

Reply:
These are not sniffed values. Source: sniffs. Here's the correct position:
```
Position: X: -2139.3203 Y: 5552.883 Z: 50.382324
Orientation: 0.261799395084381103
```

Reply:
> These are not sniffed values.

Yupe. Clearly what i thought as well. This pr is getting closed by me.